### PR TITLE
Recreate select boxes to provide better search functionality

### PR DESCRIPTION
**Pull request by HarryDavisgithub**
Changes - 
- Removed bootstrap selects for year and indicator
- Added theme to search
- Improved readability by functioning out large code blocks

### CODE REVIEW

File: world-bank-app/src/components/Search.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from "react";
-import { Form, Button } from "react-bootstrap";
+import { Form, Button, Container, Navbar } from "react-bootstrap";
```
Comment (TamoorW) [inline world-bank-app/src/components/Search.js:2]: Have you considered importing individually like 
`import Button from 'react-bootstrap/Button';`
Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client.

File: world-bank-app/src/components/Search.js [hunk 2]
```diff
@@ -28,19 +29,19 @@ export default function Search(props) {
 		for (let i = 2015; i >= 1960; i--) {
 			key++;
 			arr.push(
-				<option key={key} value={i}>
-					{i}
-				</option>
+				<Select.Option key={key} value={i.toString()}>
+					{i.toString()}
+				</Select.Option>
```
Comment (TamoorW) [inline world-bank-app/src/components/Search.js:34]: very good looking

File: world-bank-app/src/components/Search.js [hunk 3]
```diff
@@ -55,69 +56,130 @@ export default function Search(props) {
...
+			</div>
+			<Container>
+				<footer className="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
+					<p className="col-md-4 mb-0 text-muted">© 2022 Team Delta</p>
+					<div className="col-md-4 d-flex align-items-center justify-content-center mb-3 mb-md-0 me-md-auto link-dark text-decoration-none">
+						<img className="bi me-2" height="32" src={Logo} alt="world-bank-logo"></img>
+					</div>
+				</footer>
+			</Container>
+		</Container>
```
Comment (olliecase-green) [inline world-bank-app/src/components/Search.js:183]: Really good design for the indicator inputs, looks good

**Review by olliecase-green — APPROVED** [review]
Looks good

File: world-bank-app/src/components/Search.js (hunk 1 above)
Comment (HarryDavisgithub) [reply world-bank-app/src/components/Search.js:2]: Sounds good, will do that now

**Review by TamoorW — APPROVED** [review]
Very nice

**Pull request merged**